src/simtools/model/model_parameter.py

```diff
@@ -532,10 +532,11 @@ def export_nsb_spectrum_to_telescope_altitude_correction_file(self, model_direct
         model_directory: Path
             Model directory to export the file to.
         """
+        print("AAAAA", self._simulation_config_parameters)
```

Comment:
We should add a test to catch these debugging printouts =)

(This one was removed in a subsequent PR.)

Reply:
No no, here we are testing the code reviewers...